.pre-commit-config.yaml

```diff
@@ -1,6 +1,7 @@
 repos:
   - repo: https://github.com/antonbabenko/pre-commit-terraform
-    rev: v1.50.0
+    rev: v1.72.0
     hooks:
       - id: terraform_fmt
+        verbose: true
```

Comment:
often takes 20s on my machine haven't tracked down why but this at least shows the time